src/QtDBMigration.cpp

```diff
@@ -36,20 +37,27 @@ class QtDBMigrationPrivate
     bool migrate();
     bool migrate(int version);
 
+
 private:
     bool createMetaTable();
 
     QtDBMigration *m_q;
     AbstractConfig *m_config;
+    AbstractConfig *m_folder_config;
 };
 
 QtDBMigrationPrivate::QtDBMigrationPrivate(QtDBMigration *q) :
     m_q(q) ,
-    m_config(new JsonConfig(q->m_configPath))
+    m_config(new JsonConfig(q->m_configPath)),
+    m_folder_config(new FolderConfig(q->m_configPath))
```

Comment:
No.

From `q` should be known what type of config we want to have. We won't have a Json config and Folder config at the same time!

Reply:
Fixed, that should not be possible anymore.